app/build.gradle

```diff
@@ -42,6 +42,7 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:23.1.1'
     apt 'org.parceler:parceler:1.0.4'
     compile 'org.parceler:parceler-api:1.0.4'
+    compile 'com.makeramen:roundedimageview:2.2.1'
```

Comment:
Had fun with this library in a previous homework assignment I liked it too! Are you thinking about using round pet pictures in our favorites list view?